src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -926,3 +928,194 @@ def acquireTiledArea(streams, stage, area, overlap=0.2, settings_obs=None, log_p
+    :param ccd: (Camera) the camera to use for the acquisition
+    :param overlap: (0<float<1) the overlap between tiles (in percentage)
+    :param settings_obs: (dict) the settings for the acquisition
+    :param log_path: (str) path to the log file
+    :param zlevels: (list of floats) the initial zlevels to use for the stitching
+    :param registrar: (str) the type of registration to use
+    :param weaver: (str) the type of weaver to use
+    :param focusing_method: (str) the focusing method to use
+    :return: (ProgressiveFuture) an object that represents the task, allow to
+        know how much time before it is over and to cancel it. It also permits
+        to receive the result of the task, which is a list of model.DataArray:
+        the stitched acquired tiles data for each area
+    """
+    # Create a progressive future with running sub future
+    future = model.ProgressiveFuture()
+    future.running_subf = model.InstantaneousFuture()
+    future._task_lock = threading.Lock()
+
+    task = AcquireOverviewTask(streams, stage, area, focus, ccd, future, overlap, settings_obs,
+                               log_path, zlevels,
+                               registrar, weaver, focusing_method)
+    future.task_canceller = task.cancel  # let the future cancel the task
+
+    future.set_progress(end=task.estimate_time() + time.time())
+    # connect the future to the task and run in a thread
+    executeAsyncTask(future, task.run)
+
+    return future
+
+
+class AcquireOverviewTask(object):
+    """
+    Create a task to run autofocus and tiled acquisition for each area in the list of areas
+    """
+
+    def __init__(self, streams, stage, areas, focus, ccd, future, overlap=0.2, settings_obs=None, log_path=None,
+                 zlevels=None,
+                 registrar=REGISTER_GLOBAL_SHIFT, weaver=WEAVER_MEAN, focusing_method=FocusingMethod.NONE):
+        # site and feature means the same
+        self._stage = stage
+        self._future = future
+        self.streams = streams
+        self.areas = areas  # list of areas
+        self._ccd = ccd  # TODO to delete after ccd.data components is used from streams in do_autofocus_roi
+        self._focus = focus
+        self.focus_rng = self._focus.getMetadata().get(model.MD_POS_ACTIVE_RANGE, None)
+
+        if self.focus_rng is None:
+            # No absolute range is available, so use a relative range (e.g. as on METEOR
+            self.focus_rng = self._focus.getMetadata().get(model.MD_SAFE_REL_RANGE, SAFE_REL_RANGE_DEFAULT)
+            self.focus_rng = (self.focus_rng[0] + self._focus.position.value["z"],
+                              self.focus_rng[1] + self._focus.position.value["z"])
+
+        self.conf_level = 0.8
+        self.focusing_method = focusing_method
+        # The number of focus points are constant for now
+        # TODO: make it variable according to the size of the ROI, is it useful?
+        self.n_focus_points = (3, 3)
```

Comment:
This task of scaling the number of focus points with the fov is in the backlog. I have not worked on it. Is it something to prioritize?

Reply:
this is something we still need to test, could be that it's every 4 tiles, but it could also be that 3x3 tiles is always good. I think it's fine to keep it in the backlog for now, until the first testing has been done.

Reply:
Agree. Let's make it a separate task. I think that if the ROI is tiny, 3x3 is probably too many.